andela/magma-backend
# #167749972 Implement edit travel request endpoint

#### Description of Tasks to be completed

- setup test suite for edit travel request endpoint
- create a editTrip function in the RequestService class
- implements editTrip endpoint in the RequestController class
- add the route
- test with postman

#### How should this be manually tested?

- Clone the Repo, change into the directory and run `npm install`
- Run `npm test` to see all test cases
- Run `npm run dev` to start the server
- Make a `PATCH` request to `http://localhost:{PORT}/api/v1/requests/id` on Postman using valid credentials

#### Relevant pivotal tracker story

[167749972](https://www.pivotaltracker.com/story/show/167749972)
